### DISCUSSION TITLE
关于 | 作者与站点

### DISCUSSION THREAD

**Discussion by shenweiyan** [shenweiyan/Digital-Garden#4] [0.1-站点]
聊一聊关于作者和这个站点的一些事情。

## 缘起

程序员都有一个博客梦，对我来说，写写字也可以成为我调解压力的一个很好方式。

在独立博客或者说独立站点前，一直有一种 "背靠大树好乘凉" 的心理把自己的文档建立的大平台的服务上，从开源中国、蚂蚁笔记、博客园、简书、CSDN、语雀上踩过不少坑后终于认识到数据可控和 DIY 的好处，于是有了独立博客站点的想法。

静态博客站点一直是个人最情有独钟的选择，省去了服务器折腾续费运维的种种麻烦，可以更加专注于文字本身的写作。也曾经纠结过 Jekyll、Hexo、Hugo 的选型，但最终被 Hugo 极快的页面编译生成速度所折服，并深受[谢益辉](https://github.com/yihui)前期的 [ivy-hugo-theme](https://github.com/shenweiyan/ivy-hugo-theme) 博客风格影响，自己倒腾了一个 [ICS-Hugo-Theme](https://github.com/shenweiyan/ICS-Hugo-Theme) 用了好几年。

## 语雀

在独立博客站点前不得不把语雀单独拎出来说一下。

从 2018 到 2023 年，语雀一直都是我重度使用的主力平台，没有之一。我也一度以为语雀会成为了未来 5-10 年的主力平台（由于[语雀天使](https://weiyan.cc/note/2021-10-14-yuque-vip/)的福利，我的语雀会员也延长到了 **2037** 年！），[All in Yuque](https://www.yuque.com/shenweiyan/notebook/all-in-yuque) 也成为了我那一段时间的坚持。不幸的是，语雀在 2023 年发生了非常严重的[宕机事件](https://www.zhihu.com/question/627448953)，加上前一年沸沸扬扬的[收费事件](https://www.zhihu.com/question/562238887)，以及语雀固有的一些问题，对于语雀的信任开始动摇。

直至 2023 年由于各种不可描述的原因，终于把继续使用语雀的最后一根稻草压垮，逐步开始拥抱 GitHub，并在这个全球最大的开源平台上深入摸索。

## 当下

正如个人在[语雀花园](https://www.yuque.com/shenweiyan)所说，2023年7月起个人大部分文章已经转移至 [GitHub](https://github.com/shenweiyan)，并同步至 [Digital-Garden](https://github.com/shenweiyan/Digital-Garden/) (后期主要在 [Discussions](https://github.com/shenweiyan/Digital-Garden/discussions) 和 [维燕的数字花园](https://shenwy.com/) 进行写作与更新)，语雀会保持不定期更新（基本已经停更）！

现在的这个站点最开始是基于 [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/) 进行部署，后面又经历了[一些变更](https://github.com/shenweiyan/Digital-Garden/discussions/45)，但总体样式始终没有脱离 mkdocs-material 的风格。在内容上整合了个人 **[语雀知识库](https://www.yuque.com/shenweiyan)** 几个主要知识库文章，又拉取了 [Digital-Garden](https://github.com/shenweiyan/Digital-Garden/) 的 discussions 文章，汇总成现在的独立站点。

## 未来

我希望 [WeiYan.CC](https://weiyan.cc/)（2025 年 9 月 17 换成了 [ShenWY.com](https://shenwy.com/)）能成为我知识管理的最后一站，虽然未来这些文章展现的形式，或者选择用于部署的技术有所改变，但至少域名是不变的（除非我能有幸注册到 com/cn/net 其中之一的主流域名），通过这个域名你至少还能找到曾经我写下的那些字。

不忘初心，方得始终，希望下一个十年，还能在这里遇到一个曾经的我，看到曾经的文字。

## 作者

沈维燕（史提芬先森/章鱼猫先生），一个 90 后的广东人，熟悉粤语、国语，略懂英语。

- 毕业于南方医科大学（原中国人民解放军第一军医大学）基础医学院生物信息学专业。
- 现工作生活于广州，主要从事 BIO & IT 的一些相关工作。
- 平时喜欢逛逛技术论坛，玩玩羽毛球，看看电影，瞎折腾一下技术。
- 不务正业之余喜欢记录一些生活工作学习中的一些想法。
- 乐于分享，喜欢把事情简单化，程序化。

## 联系

个人目前用的比较多的沟通工具，一个是**邮箱**，另外一个是**微信**，你可以通过这两种方式直接和我联系。

- 邮箱：<shenweiyan@aliyun.com>
- 微信：ishenweiyan（添加微信好友，请注明真实姓名）

!!! note "请备注真名实姓，让我感受到一个真实的人的气息"
    如果给我发**邮件**，或者通过**微信添加好友**，请写上您的**真名实姓**，让我感受到一个**真实的人**的气息。我不太愿意跟**网名**打交道，对于那些不知来路、上来就问问题的微信和邮件，我通常会直接忽略。

**Comment by chuxinyuan** [comment]
> 如果给我发邮件，或者通过微信添加好友，请写上您的真名实姓，让我感受到一个真实的人的气息。我不太愿意跟网名打交道，对于那些不知来路、上来就问问题的微信和邮件，我通常会直接忽略。

你这个要求很谢益辉嘛

**Reply by shenweiyan** [reply]
感谢益辉大大说出了我们一直想说的话。

**Comment by fcatat** [comment]
我前两天刚好刷到你的网站。从https://nops.icu 留言过来

**Reply by shenweiyan** [reply]
看了博主站点的很多 DeepSeek 和运维的知识，也是获益良多👍
